db/migrations/20200512141017_remove_urn_state.sql

```diff
@@ -0,0 +1,232 @@
+-- +goose Up
+-- SQL in this section is executed when the migration is applied.
+DROP FUNCTION IF EXISTS api.all_urn_states;
+
+DROP FUNCTION IF EXISTS api.get_urn;
+CREATE FUNCTION api.get_urn(ilk_identifier text, urn_identifier text, block_height bigint DEFAULT api.max_block()) RETURNS api.urn_snapshot
+    LANGUAGE sql STABLE STRICT
+    AS $_$
+
+SELECT *
+    FROM api.all_urns(get_urn.block_height)
```

Comment:
Seems kinda heavyweight to query `all_urns` for one urn - might it be better to:
```
SELECT * FROM api.urn_snapshot WHERE ilk_identifier = x AND urn_identifier = y AND block_height <= n ORDER BY block_height DESC LIMIT 1
```

Reply:
I was trying to be consistent with the other ones that use all_urns, but you're probably right.